services/app/karma.conf.js

```diff
@@ -1,31 +1,30 @@
 const webpackConfig = require('./webpack.config');
-const webpack = require('webpack');
 
 webpackConfig.module.rules.unshift({
   test: /isIterable/,
   loader: 'imports?Symbol=>false'
 });
+webpackConfig.entry = undefined;
+
+// Remove hot reload
+webpackConfig.plugins = [webpackConfig.plugins[webpackConfig.plugins.length-1]];
```

Comment:
Would it be possible to be more principled about this, ie to explicitly remove the hot reload plugin rather than assume it's position?

Reply:
Made an attempt at this :) hard to do when the copy plugin has poor naming though.